### ISSUE TITLE
Suggestion for README installation notes

### ISSUE THREAD

**Issue by Ciantic** [BlackReloaded/wsl2-ssh-pageant#14]
I have a suggestion, make the installation snippet in README like this: 

```shell
sudo apt-get install socat
cd ~/.ssh/
wget https://github.com/BlackReloaded/wsl2-ssh-pageant/releases/latest/download/wsl2-ssh-pageant.exe
chmod +x wsl2-ssh-pageant.exe
```

(Notice that URL link is to *latest* release)

Exact lines does not matter, but grand idea is: It would allow copy pasting the whole thing and running in one swoop.

Thanks, great utility.

**Comment by tobiaskohlbau** [comment]
I'm fine with that change. Maybe in addition to the already existing option like an _Getting started/fast onboarding_ option.

What are your thoughts @BlackReloaded?

@Ciantic if accepted would you like to open a PR?

**Comment by Ciantic** [comment]
Of course. I modified the snippet a little, being more explicit:

```
sudo apt-get install socat
wget -o "$HOME/.ssh/wsl2-ssh-pageant.exe" https://github.com/BlackReloaded/wsl2-ssh-pageant/releases/latest/download/wsl2-ssh-pageant.exe
chmod +x "$HOME/.ssh/wsl2-ssh-pageant.exe"
```

**Comment by BlackReloaded** [comment]
I'm totaly fine with that.

**Comment by tobiaskohlbau** [comment]
Closed by #20
